Resources/Data Sets/index.json

```diff
@@ -775,6 +775,20 @@
           "kind": "file",
           "associated_datafile": "Anscombe's Quartet.csv"
         },
+        {
+          "name": "The Datasaurus Dozen (Long)",
```

Comment:
This will need to be added to the Dutch and German index.json files

Reply:
wait, why aren't those files translated in a similar way with gettext? I just noticed that the jasp files are completely missing if you change the language, instead of using the English name/ description as a fall back.

If it's just because it's a JSON file, then we should maybe set that up differently.

Reply:
They will be rewritten as .qml, but until that happens we need to manually keep track of changes

Reply:
If someone helps me translating the description field do Dutch, I will add it. I assume correctly that I don't need to translate the names of the files though, right?

Reply:
Actually the description file needs some improvement, let me fix that first